### PR TITLE
[Pushover Action] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: bundles/action/org.openhab.action.pushover/README.md [hunk 1]
```diff
@@ -4,46 +4,49 @@ The Pushover action service allows you to notify mobile devices of a message usi
...
-| defaultUrlTitle | | No | URL title to attach to the message if not specified in the command. This can be used to trigger actions on the device. |
-| defaultRetry | 300 | No | When priority is 2 (high priority), how often (in seconds) should messages be resent |
-| defaultExpire | 3600 | No | When priority is 2 (high priority), how long (in seconds) to continue resending messages until acknowledged |
+You may define default values for parameters to action calls in the file `services/pushover.cfg`.
+None of the configuration parameters are required as you can specify required configuration items in the action call, but you must at least provide an *API Token*, *User/Group Key* and a *Message* in some manner before a message can be pushed.
+
+| Property        | Default | Required                                          | Description                                                                                                            |
+|-----------------|---------|-:-:-----------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
```
Comment (9037568) [inline bundles/action/org.openhab.action.pushover/README.md]: This is broken.

Comment (bgilmer77) [reply bundles/action/org.openhab.action.pushover/README.md]: Hi @9037568.  I believe my latest commit fixes this.  Please look, and if okay, please merge.

Comment (9037568) [reply bundles/action/org.openhab.action.pushover/README.md]: Sorry, the last commit doesn't appear to have changed anything.

**Comment by 9037568** [comment]
Thanks, @bgilmer77 !

**Pull request merged**